File: rolemadelen/baekjoon/step9/11653.rs
use std::io::{stdin};

fn read() -> i32 {
    let mut input = String::new();
    stdin().read_line(&mut input).unwrap();
    input.trim().parse().unwrap()
}

fn main() {
    let x = read();
    let mut x = x;

    while x > 1 {
        let mut target = 2;

        while x % target != 0 {
            target += 1;
        }
        x /= target;

        println!("{target}");
    }
}

File: xyident/baekjoon/9단계/11653___.js
// 시간초과

let input = require('fs').readFileSync('/dev/stdin').toString().trim();
// let input = 72;
if(input != 1){
  while(input > 1){
    for(let i = 2; i < input; i++){
      if(input % i == 0){
        console.log(i);
        input /= i;
        break;
      }
    }
  }
}